### DISCUSSION TITLE
Where is "my" GH homepage with events for my repos?

### DISCUSSION THREAD

**Discussion by janbrasna** [community/community#66363] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

OK, for some time I was forced to click on a link to see activity relevant to me (my teams, products, business, repos, wikis… basically making a living), with the new facespace snapstagram social feed of random stuff I don't want to see as first thing popping up more and more often (as a way to push this "great" idea from gh product team over the last two years in various beta stages etc.) but still I was always able to go back to timeline of updates what I needed, not this noise, by clicking somewhere. Today I've logged in, and **there's no way to view only «events happening on things "we own"…»**

Even if I filter out noise junk given the shiny new redesign notification box instructions… I still can't differentiate between fun community randomness… vs. my own livelihood. Every checkbox item ends up with _"… from repositories"_. **From what repositories?** I need to be able to state from what repositories.
- I viewed or liked? 
- Your AI thinks are just cool given my behav data? 
- I interacted with, have issues/PR in, or have forks of? 
- I have starred? 
- I committed to? 
- I maintain? 
- I own? 
- My organisations own?

There's whole lotta difference between getting work done on my own commercial products with my teams vs. randomly browsing what's new in the foss space out of boredom… but my GH home forces me to do both now at the same time. WTH? Even after a year+ of feedback you still force push this to people not wanting any of this?

**Comment by janbrasna** [comment]
FWIW no combination of checkboxes shows me the same timeline I need to see, the one that was available for years. Wiki edits, triage tag changes etc. … still figuring out what's being left out in the new view, with no filters to get to such content… :/

**Comment by mikesprague** [comment]
In full agreement and also posted about this yesterday. You've prob already seen this but just in case...

https://github.com/dashboard-feed

No guarantee it will stick around (and full width with no margins is a bit much) but you can currently view that content at least.

**Reply by janbrasna** [reply]
Yup, just found it in the threads around… Thanks Mike! Hopefully it helps other desperate users in need to check their daily updates from orgs/repos without wasting precious time trying to navigate the gamification&socmed mess. 

It only shows the true size of the insult: events I can't filter to in the new system are still present in the old feed. That tells me the new feed lost some bits of the event types it can load. Crap:/

(Honestly this whole discussion space as a destination from all the "Give feedback" links w/o any context is a proper hell in its own… Not sure if everyone is supposed to provide feedback of their own, or we're supposed to find "preview" threads from GH staff from yesteryear to add to, or look for a launch announcement and chip in under there… I'm always lost when a feature/preview leads me here to "give feedback"… only to be welcomed at a discussion category overview that looks like some kid's social feed exploded with emoji everywhere, and no GH staffers in sight to even nod that someone's listening here or acknowledging the reported problems, so you're not just another desperate user left in a pool of other desperate users talking to each other… [As none of the bugs/feedback I've ever reported here got any attention of anyone else than just regular users like me who run into the same thing and ended up in my thread, with no sign of GH to be taking any care of anything that ends up written around here. So it's a vocal community… that's supposed to comfort themselves? The time and effort to "provide feedback" as directed in many parts of the UI doesn't seem to be acted upon by anyone from the GH staff, which is rather sad. Even a checkmark "yea we've seen this, the effort reporting this was not wasted" would be golden…]

**Reply by mikesprague** [reply]
We're on the same page. Rollout, reporting via this discussion space, and even lack of communication since has been lousy at best. Hopefully they are paying attention to the pushback.
